7_react/react1/week2/my-app/src/TodoList/TodoList.js

```diff
@@ -0,0 +1,93 @@
+import TodoItem from "../TodoItem/TodoItem"
+import "./TodoList.css"
+import React, { useEffect, useState } from "react"
+
+
+const defaultTodoList = [{
+    id: new Date().getTime() + 1,
+    description: "Get out of bed",
+    deadline: "Wed Sep 13 2017",
+    done: false
+},
+{
+    id: new Date().getTime() + 2,
+    description: "Brush teeth",
+    deadline: "Thu Sep 14 2017",
+    done: false
+
+},
+{
+    id: new Date().getTime() + 3,
+    description: "Eat breakfast",
+    deadline: "Fri Sep 15 2017",
+    done: false
+}]
+
+function TodoList() {
+    const [todoList, setTodoList] = useState(defaultTodoList);
+    const [timer, setTimer] = useState(0);
+
+    const updateTime = () => {
+       setTimer((prev) => {
+            return prev+1;
+       })
+    }
+  
+    useEffect(() => {
+        const interval = setInterval(() => updateTime(), 1000);
+
+        return () => clearInterval(interval);  
+    }, [])
+
+    const addNewTask = () => {
+        const newRandomTask = {
+            id: new Date().getTime(),
+            description: 'Random task',
+            deadline: 'Wednesday Oct 12 2024',
```

Comment:
nit-pick: If you are already dealing with a date, we can use this:
`deadline: new Date('Wednesday Oct 12 2024'),`